### PR TITLE
update documentation

### LINKED ISSUE #11: Document deployment and usage on arbitrary cluster
Currently, we document only usage on local kubevirtci cluster. We should add a description how to install multus and ovs-cni on a cluster and how to use it. This depends on #10

### PR DESCRIPTION
**Pull request by phoracek**
Resolves #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,48 @@
 # Open vSwitch CNI plugin
 
+This plugin allows user to define Kubernetes networks on top of Open vSwitch bridges available on nodes. IPAM is currently not supported. There is no scheduling involved, desired bridges must be precreated on all nodes. Also, ovs-cni does not configure bridges, it's up to user to connect them to L2, L3 or an overlay network. Finally please note that Open vSwitch must be installed and running on the host.
```
Comment (SchSeba) [inline README.md]: Maybe add here that also the network attachment CRD needs to be applied and multus cni plugin configured on every node in the cluster.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,48 @@
...
+
+## Usage
+
+First define network definition. This object specifies to which Open vSwitch bridge should be pod attached and what VLAN ID should be set on the port. For more information, check [plugin documentation](docs/cni-plugin.md).
```
Comment (yuvalif) [inline README.md]: "First define network definition"->"First create network attachment definition"

Comment (yuvalif) [reply README.md]: "should be pod"->"should the pod be"

File: README.md [hunk 3]
```diff
@@ -1,5 +1,48 @@
...
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: ovs-conf
+spec:
+  config: '{
+      "cniVersion": "0.3.1",
+      "type": "ovs",
+      "bridge": "br1",
+      "vlan": 100
```
Comment (yuvalif) [inline README.md]: is vlan optional? can we define a port without specifying a vlan?

File: README.md [hunk 4]
```diff
@@ -1,5 +1,48 @@
...
+  name: samplepod
+  annotations:
+    k8s.v1.cni.cncf.io/networks: ovs-conf
+spec:
+  containers:
+  - name: samplepod
+    command: ["/bin/bash", "-c", "sleep 99999"]
+    image: alpine
+EOF
+```
```
Comment (yuvalif) [inline README.md]: can you please add "ip link.." command to execute on the pod to verify that the interface was created

File: README.md (hunk 1 above)
Comment (yuvalif) [inline README.md]: could you also add sample command to create the ovs bridge?

Comment (yuvalif) [inline README.md]: at the end of the readme (appendix?) may be interesting to add example of manual configuration of host labels to allow for host selection by the pod

File: README.md (hunk 3 above)
Comment (phoracek) [reply README.md]: It's described in "For more information, check [plugin documentation](docs/cni-plugin.md)." Should I state it also here? I don't want to get into more and more details on from README.md as we add other paramters.

File: README.md (hunk 1 above)
Comment (phoracek) [reply README.md]: It is already in docs/local-cluster-example.md, I will add it to detailed usage example for arbitrary cluster too. I'd prefer the from README.md to be as clean as possible.

Comment (phoracek) [reply README.md]: I will create another documentation file with this under docs/.

**Comment by phoracek** [comment]
@SchSeba @yuvalif I reworked the documentation. Could you please take a look? Thanks

File: README.md (hunk 1 above)
Comment (yuvalif) [reply README.md]: "local cluster" is for developers. the README should be for users.
if you want to keep it clean, maybe refer to "getting started" page?

Comment (yuvalif) [reply README.md]: covered in: deployment-on-arbitrary-cluster.md

**Pull request merged**